docs/Architecture/miscellaneous-components.md

```diff
@@ -18,6 +18,8 @@ PostgreSQL is an open source object-relational database system. Paralus uses Pos
 
 Elastic search is an open source distributed, search and analytics engine for various types of data. Paralus uses elastic search to store all the system and kubectl logs that are generate. These logs persist in the database even after the pod is killed or restarts.
 
+> Note: This is now an optional component from release 0.1.9.
```

Comment:
Filebeat is also an optional component. only required with elasticsearch

Reply:
Do we need to add Fluentbit as well here in this misc components? If yes, I'll add a generic statement about it, can you provide the specific reason it is used here? @niravparikh05

Reply:
yeah, we would need to add that as well

Reply:
Added fluentbit.